guides/common/modules/proc_configuring-dynamic-dns-update-with-gss-tsig-authentication.adoc

```diff
@@ -138,24 +138,11 @@ grant {smart-proxy-principal}\047__{foreman-example-com}@EXAMPLE.COM__ wildcard
 .Configuring the {Project} or {SmartProxyServer} that manages the DNS service for the domain
 
 . Use the `{foreman-installer}` command to configure the {Project} or {SmartProxy} that manages the DNS Service for the domain:
-* On {Project}, enter the following command:
+* On {Project} or {SmartProxy}, enter the following command:
```

Comment:
I wasn't sure how to write these instructions, but they feel too complex with the simplification.